Extension_CloudFlare_Page_View.js

```diff
@@ -33,7 +33,7 @@ jQuery(function($) {
 
 
 		.on('click', '.w3tc_cloudflare_zone_page', function() {
-			var page = jQuery(this).attr('data-page');
+			var page = jQuery(this).prop('data-page');
```

Comment:
That and all similar looks like a bug.
prop shouldnt return value for non-DOM properties.

Did you check it works?

Reply:
I'm fixing all of those "data-" references to see "data()", which works.